### PR TITLE
feat: support description with newline

### PR DESCRIPTION
**Pull request by hykuan**
swag will treat /n as /(92) n(110) not LF(10), so i replace it.

and i use gopkg.in/yaml.v2 which can generated correct yaml format with newline

### CODE REVIEW

**Comment by hykuan** [comment]
what should i do to make CI pass ?

File: example/basic/docs/docs.go [hunk 1]
```diff
@@ -188,13 +188,13 @@ var doc = `{
         "web.APIError": {
             "type": "object",
             "properties": {
-                "CreatedAt": {
+                "createdAt": {
```
Comment (easonlin404) [inline example/basic/docs/docs.go]: Cannot change to Lower case begin.

File: example/basic/docs/docs.go [hunk 2]
```diff
@@ -188,13 +188,13 @@ var doc = `{
...
                     "type": "string"
                 },
-                "ErrorCode": {
+                "errorCode": {
```
Comment (easonlin404) [inline example/basic/docs/docs.go]: Same as above

File: example/basic/docs/docs.go [hunk 3]
```diff
@@ -188,13 +188,13 @@ var doc = `{
...
                     "type": "integer"
                 },
-                "ErrorMessage": {
+                "errorMessage": {
```
Comment (easonlin404) [inline example/basic/docs/docs.go]: Same as above

**Comment by hykuan** [comment]
OK, you can close the PR, and i will wait someone implements the feature.

**Comment by easonlin404** [comment]
@hykuan PR is welcome.

**Pull request closed without merging**